test/spawn.jl

```diff
@@ -415,11 +420,14 @@ end
 @test Base.shell_split("\"\\\\\"") == ["\\"]
 
 # issue #13616
-@test_throws ErrorException collect(eachline(pipeline(`$catcmd _doesnt_exist__111_`, stderr=devnull)))
+let p = eachline(pipeline(`$catcmd _doesnt_exist__111_`, stderr=devnull))
+    @test_throws(ErrorException("failed process: Process(`cat _doesnt_exist__111_`, ProcessExited(1)) [1]"),
```

Comment:
@vtjnash This needs to be ``@test_throws(ErrorException("failed process: Process(`$catcmd``, otherwise the error message doesn't match up.